README.md

```diff
@@ -178,10 +178,10 @@ For more information about contributing to the Kubernetes documentation, see:
 | [Italian](README-it.md)    | [Ukrainian](README-uk.md)  |
 | [Japanese](README-ja.md)   | [Vietnamese](README-vi.md) |
 
-## Code of conduct
+## Code of Conduct
```

Comment:
Please refer to the[ Kubernetes style guide](https://kubernetes.io/docs/contribute/style/style-guide/#headings), sentence case is used for headings
```suggestion
## Code of conduct
```
Please revert